python/llm/dev/benchmark/all-in-one/run.py

```diff
@@ -942,9 +985,10 @@ def run_transformer_autocast_bf16( repo_id,
                         in_out_pairs.remove(in_out)
             run_model(model, api, in_out_pairs, conf['local_model_hub'], conf['warm_up'], conf['num_trials'], conf['num_beams'],
                       conf['low_bit'], conf['cpu_embedding'], conf['batch_size'])
-        df = pd.DataFrame(results, columns=['model', '1st token avg latency (ms)', '2+ avg latency (ms/token)', 'encoder time (ms)',
-                                            'input/output tokens', 'actual input/output tokens', 'num_beams', 'low_bit', 'cpu_embedding', 
-                                            'peak mem (GB)'])
+        if api!="transformer_int4_gpu":
+            df = pd.DataFrame(results, columns=['model', '1st token avg latency (ms)', '2+ avg latency (ms/token)', 'encoder time (ms)',
+                                                'input/output tokens', 'actual input/output tokens', 'num_beams', 'low_bit', 'cpu_embedding',
+                                                'peak mem (GB)'])
 
-        df.to_csv(f'{current_dir}/{api}-results-{today}.csv')
+            df.to_csv(csv_name)
```

Comment:
Does it work to keep the results and write to csv again for transformer_int4_gpu as well?

Reply:
I think it may work.